ui/app/templates/vault/cluster/access/method/item/list.hbs

```diff
@@ -0,0 +1,87 @@
+<PageHeader as |p|>
+  <p.top>
+    {{#key-value-header path="vault.cluster.access.methods"}}
+      <li>
+        <span class="sep">
+          /
+        </span>
+        {{#link-to "vault.cluster.access.methods"}}
+          methods
+        {{/link-to}}
+      </li>
+    {{/key-value-header}}
+  </p.top>
+  <p.levelLeft>
+    <h1 class="title is-3">
+      {{capitalize method}}
```

Comment:
Do we want to add an Icon above this? - should this be `method` or the `path` for the auth method?

Reply:
So I think this should be id of the method